.travis/README.md

```diff
@@ -0,0 +1,201 @@
+
+## How to Configure Travis CI to Deploy Your Application to Fly
+
+In this tutorial, you’ll learn how to deploy an application to Fly.io using Travis CI. You’ll see how to connect your application’s repository, and walk through the steps to configure Travis CI to build and deploy your application to Fly.
+
+### Prerequisites
+
+- A [GitHub](https://github.com/) or [Bitbucket](https://bitbucket.org/) account.
+- The [Flyctl command line tool](https://fly.io/docs/flyctl/installing/).
+- (Optional) [Docker](https://www.docker.com/) for testing the demo application locally.
+
+### Cloning the Sample Application
+
+To demonstrate deploying an application to Travis, you can use the [Flygreeting sample app on GitHub](https://github.com/fly-examples/flygreeting).
+
+Fork [flygreeting](https://github.com/fly-examples/flygreeting) to your own repository (if you use GitHub), then clone it to your local machine:
+
+```
+git clone git@github.com:fly-examples/flygreeting.git
+cd flygreeting
+```
+
+If you use Bitbucket, you can `git push` the cloned repository to your Bitbucket account.
+
+To test the Flygreeting app locally, you can run it using Docker:
+
+```
+docker build -t flygreeting .
+docker run -p 8000:8080 -d flygreeting
+```
+
+To test the application, call the `/countries` endpoint using curl:
+
+```bash
+curl http://localhost:8000/v1/countries/
+```
+
+You should see a JSON response with all the available country abbreviations:
+
+```json
+{"countries":["AD","AE","AF","AG","AI","AL","AM","AO","AQ","AR","AS","AT","AU","AW","AX","AZ","BA","BB","BD","BE","BF","BG","BH","BI","BJ","BL","BM","BN","BO","BQ","BR","BS","BT","BV","BW","BY","BZ","CA","CC","CD","CF","CG","CH","CI","CK","CL","CM","CN","CO","CR","CU","CV","CW","CX","CY","CZ","DE","DJ","DK","DM","DO","DZ","EC","EE","EG","EH","ER","ES","ET","FI","FJ","FK","FM","FO","FR","GA","GB","GD","GE","GF","GG","GH","GI","GL","GM","GN","GP","GQ","GR","GS","GT","GU","GW","GY","HK","HM","HN","HR","HT","HU","ID","IE","IL","IM","IN","IO","IQ","IR","IS","IT","JE","JM","JO","JP","KE","KG","KH","KI","KM","KN","KP","KR","KW","KY","KZ","LA","LB","LC","LI","LK","LR","LS","LT","LU","LV","LY","MA","MC","MD","ME","MF","MG","MH","MK","ML","MM","MN","MO","MP","MQ","MR","MS","MT","MU","MV","MW","MX","MY","MZ","NA","NC","NE","NF","NG","NI","NL","NO","NP","NR","NU","NZ","OM","PA","PE","PF","PG","PH","PK","PL","PM","PN","PR","PS","PT","PW","PY","QA","RE","RO","RS","RU","RW","SA","SB","SC","SD","SE","SG","SH","SI","SJ","SK","SL","SM","SN","SO","SR","SS","ST","SV","SX","SY","SZ","TC","TD","TF","TG","TH","TJ","TK","TL","TM","TN","TO","TR","TT","TV","TW","TZ","UA","UG","UM","US","UY","UZ","VA","VC","VE","VG","VI","VN","VU","WF","WS","YE","YT","ZA","ZM","ZW"]}
+```
```

Comment:
Updated the instructions to use curl and show the response as suggested by @codepope.